tun/src/unix/linux/mod.rs

```diff
@@ -50,6 +50,12 @@ impl TunInterface {
         iff.ifr_name = string_to_ifname(&self.name()?);
         iff
     }
+    
+    #[throws]
+    fn in6_ifreq(&self) -> sys::in6_ifreq {
+        let mut iff: sys::in6_ifreq = unsafe { mem::zeroed() };
+        iff.ifr6_ifindex = 0;
```

Comment:
@conradev obviously it doesn't make too much sense to set this to 0 - what should I set it to?

Reply:
@sampoder `self.index()?;` 😄

Reply:
🤦 sleepy me!